packages/cactus-test-geth-ledger/src/test/typescript/integration/contractData.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
@rwat17 Do you still need this and with this file name or was it left in the diff by mistake?

Reply:
yup that was a mistake, forgot to delete this file :)